contents/docs/session-replay/privacy.mdx

```diff
@@ -94,4 +99,67 @@ To do so, you should add the CSS class name `ph-no-capture` to elements which sh
 <div class="ph-no-capture">I won't be captured at all!</div>
 ```
 
-Note that adding `ph-no-capture` will also prevent any [autocapture](/docs/data/autocapture) events from being captured from that element.
\ No newline at end of file
+Note that adding `ph-no-capture` will also prevent any [autocapture](/docs/data/autocapture) events from being captured from that element.
+
+
+# Common example configs
+
+## Maximum privacy - mask everything
+
+```
+{
+    maskAllInputs: true,
+    maskTextSelector: "*"
+}
+```
+
+## Good privacy - try to mask all email / password related things
+
+To strike a balance of usefulness versus privacy friendly, you can choose to only mask content that looks like it contains an email or password.
+With passwords, it is important to note that typically the "click to show password" buttons, turn the input type to `text` which would then reveal the value. If we check for something else like the `id` field then we can ensure that it is never captured (you of course need to ensure this field is added)
```

Comment:
```suggestion
You can mask content that looks like it contains an email or password.

For passwords, it is important to note that "click to show password" buttons typically turn the input type to `text`. This would then reveal the password. Thus instead of checking for `type='password'`, you need to check a different field, like `id`:
```